sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2949,10 +2949,10 @@ object functions {
   def unix_timestamp(s: Column, p: String): Column = withExpr { UnixTimestamp(s.expr, Literal(p)) }
 
   /**
-   * Converts to a timestamp by casting rules to `TimestampType`.
+   * Converts to a timestamp by casting rules to `TimestampType` in the seconds precision.
```

Comment:
Let's not touch this.

Reply:
May I ask you to explain why we shouldn't clarify function's behavior here?

Reply:
Of course, you can do whatever you want with much more details, but I want to minimize the change.

Reply:
Usually it's better to make the changes minimised if we have to do it to old branches to reduce the potential conflicts when backporting. Seems this is the case.